internal/common/base_client.go

```diff
@@ -0,0 +1,43 @@
+package common
+
+import (
+	"fmt"
+	"log"
+
+	"github.com/SOMAS2020/SOMAS2020/internal/common/shared"
+)
+
+const id = 420
+
+func NewClient() Client {
```

Comment:
should it not populate BaseClient correctly? `c.logf` would then safely log with the correct id. `NewClient` should therefore take in `id`

Reply:
Perhaps would even mean that the team clients don't need to impl logf at all?

Reply:
correct! :)

Reply:
So the teams' client doesnt need the clientID any more right cause its in the baseClient?